### PR TITLE
Add engine-config role

### LINKED ISSUE #47: Add role to tune engine using engine-config

### PR DESCRIPTION
**Pull request by lukas-bednar**
Fix #47

### CODE REVIEW

**Comment by lukas-bednar** [comment]
not ready for merge yet, just FYI ...

**Comment by lukas-bednar** [comment]
@StLuke @pkubica  now it is ready

File: roles/ovirt-engine-config/tasks/main.yml [hunk 1]
```diff
@@ -0,0 +1,35 @@
+---
+- name: Tune oVirt engine with engine-config, version specific
+  shell: "engine-config -s {{ item.key }}='{{ item.value }}' --cver={{ item.version }}"
+  with_items: "{{ ovirt_engine_config | list }}"
+  when: ovirt_engine_config is defined and item.version != "general"
```
Comment (StLuke) [inline roles/ovirt-engine-config/tasks/main.yml]: if more than one property is changed, move it to property file and diff with current options -> change only required

File: roles/ovirt-engine-config/tasks/main.yml [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+
+- name: Tune oVirt engine with engine-config, regardless version
```
Comment (StLuke) [inline roles/ovirt-engine-config/tasks/main.yml]: set item.version to default general, overwritten by user input if set, this second can be removed

Comment (lukas-bednar) [inline roles/ovirt-engine-config/tasks/main.yml]: the problem is that "engine-config --cver general" complains that 'general' is not valid parameter for --cver option.

**Comment by lukas-bednar** [comment]
@StLuke @pkubica any additional comments ?

**Comment by StLuke** [comment]
+1

**Pull request merged**